subsys/bluetooth/controller/Kconfig.split

```diff
@@ -0,0 +1,580 @@
+# Zephyr Bluetooth Controller configuration options
```

Comment:
Can we rename this to `Kconfig.ll_sw_split` ?

Reply:
Sure, no problem

Reply:
Done